src/lib/shared/button/LogOutButton.svelte

```diff
@@ -1,19 +1,21 @@
 <script>
 	import { session as SessionStore } from '$app/stores';
-	import { userStore } from '$lib/stores/user';
 	import { createEventDispatcher } from 'svelte';
+	import { messageStore } from '$lib/stores/message';
+	import { goto } from '$app/navigation';
 
 	const dispatcher = createEventDispatcher();
 
 	function logout() {
 		return fetch('/account/uitloggen.json', { method: 'POST' })
 			.then(() => SessionStore.set({ authenticated: false }))
-			.then(() => userStore.clearUser())
-			.then(() => dispatcher('logout'));
+			.then(() => dispatcher('logout'))
```

Comment:
Je hoeft denk ik geen `dispatcher` te doen als je daar niet naar luistert in `gymles` [route](https://github.com/iSirThijs/jungle-gym/blob/058004abd13f3526a654b9bd16c4bec221687220/src/routes/gymles/index.svelte#L53)